plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -0,0 +1,146 @@
+module Kubernetes
+  class ReleaseDoc < ActiveRecord::Base
+    self.table_name = 'kubernetes_release_docs'
```

Comment:
There's a `main_app` object that you call those helpers on. It's just that those helpers aren't included in all your controllers.